AppServer/google/appengine/tools/devappserver2/server.py

```diff
@@ -34,11 +35,12 @@
 import urlparse
 import wsgiref.headers
 
+
 from google.appengine.api import api_base_pb
 from google.appengine.api import apiproxy_stub_map
 from google.appengine.api import appinfo
 from google.appengine.api import request_info
-from google.appengine.api.logservice import log_service_pb
+from google.appengine.api.logservice import logging_capnp
```

Comment:
I made it a symlink, but @whoarethebritons replaced that through copying the file from the ~/LogService directory in the setup procedure.